sql/src/main/java/io/crate/analyze/AlterTableAnalyzer.java

```diff
@@ -56,6 +57,23 @@ public AlterTableAnalyzedStatement analyze(AlterTable node, Row parameters, Sess
         return new AlterTableAnalyzedStatement(docTableInfo, partitionName, tableParameter, table.excludePartitions());
     }
 
+    AlterTableRenameAnalyzedStatement analyzeRename(AlterTableRename node, String defaultSchema,
```

Comment:
isn't the defaultSchema part of the sessionContext? I'd either require only the sessionContext or replace the sessionContext with user so that both dependencies are explicit, but not a mixture.